rustls/src/crypto/aws_lc_rs/mod.rs

```diff
@@ -50,15 +50,12 @@ pub fn default_provider() -> CryptoProvider {
 }
 
 fn default_kx_groups() -> Vec<&'static dyn SupportedKxGroup> {
-    #[cfg(feature = "fips")]
-    {
-        DEFAULT_KX_GROUPS
-            .iter()
-            .filter(|cs| cs.fips())
-            .copied()
-            .collect()
-    }
-    #[cfg(not(feature = "fips"))]
+    // NOW REMOVED FROM THIS FORK:
+    // #[cfg(... "fips")]
+    // {
+    //     ...
+    // }
```

Comment:
minor nit:

```suggestion
    // ...
```

(I don't feel 100% strongly about this one. I do really like the idea of showing the structure of the code that was removed, which may make it a little easier to restore if needed someday in the future. But this update would make things a little more compact.)